>> anaconda-3-5.3.1 has been loaded.
>> python-3.8.2 has been loaded.
>> cuda-10.1_cudnn-7.6.5 has been loaded.
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 276, in program_main
    continue_create_empty_yaml(newloadfile_name)
  File "main.py", line 39, in continue_create_empty_yaml
    ga_params = conf_load(savefile_name)
  File "main.py", line 15, in conf_load
    return ga_params
UnboundLocalError: local variable 'ga_params' referenced before assignment
srun: error: saliksik-gpu-05: task 7: Exited with exit code 1
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 276, in program_main
    continue_create_empty_yaml(newloadfile_name)
  File "main.py", line 42, in continue_create_empty_yaml
    for i in range(run_start, ga_params['run_total'] + ga_params['runs_pending']):
KeyError: 'runs_pending'
srun: error: saliksik-gpu-05: task 6: Exited with exit code 1
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 276, in program_main
    continue_create_empty_yaml(newloadfile_name)
  File "main.py", line 42, in continue_create_empty_yaml
    for i in range(run_start, ga_params['run_total'] + ga_params['runs_pending']):
KeyError: 'runs_pending'
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 276, in program_main
    continue_create_empty_yaml(newloadfile_name)
  File "main.py", line 42, in continue_create_empty_yaml
    for i in range(run_start, ga_params['run_total'] + ga_params['runs_pending']):
KeyError: 'runs_pending'
Traceback (most recent call last):
  File "main.py", line 303, in <module>
    main()
  File "main.py", line 301, in main
    program_main(sys.argv)
  File "main.py", line 276, in program_main
    continue_create_empty_yaml(newloadfile_name)
  File "main.py", line 42, in continue_create_empty_yaml
    for i in range(run_start, ga_params['run_total'] + ga_params['runs_pending']):
KeyError: 'runs_pending'
srun: error: saliksik-gpu-05: tasks 0-1,4: Exited with exit code 1
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** STEP 237047.2 ON saliksik-gpu-05 CANCELLED AT 2021-07-14T02:38:43 ***
slurmstepd: error: *** JOB 237047 ON saliksik-gpu-05 CANCELLED AT 2021-07-14T02:38:43 ***
